docs/getting-started.md

```diff
@@ -2,12 +2,75 @@
 title: Getting started
 ---
 
+## Prerequisites
+
+Before you begin, make sure your development environment includes Node.js® and an npm package manager.
```

Comment:
...and you've turned on your computer 😆 Come on, it's not a framework for beginners that need to lead them by the hand.